test/e2e/imports.test.js

```diff
@@ -177,59 +147,6 @@ test('GET /imports/progress/:importId when import is already completed returns s
   t.same(messages2[0], messages1[messages1.length - 1])
 })
 
-// This tests that the server can force an eventsource client to disconnect by
-// responding with a 204 status code. This is in case the client does not close
-// the eventSource (as it should) after the 'complete' message is received
-test('GET /imports/progress/:importId - EventSource forced to close after single message if import has already completed', async (t) => {
```

Comment:
This test seemed nearly identical to the one above _and_ it wasn't working, so I removed it entirely.

Reply:
huh weird - but yeah, seems very similar 😄